augur/filter.py

```diff
@@ -430,3 +446,72 @@ def numeric_date(date):
         return float(date)
     except ValueError:
         return treetime.utils.numeric_date(datetime.date(*map(int, date.split("-", 2))))
+
+
+class TooManyGroupsError(ValueError):
+    ...
```

Comment:
Could you use`pass` here instead the ellipsis equivalent? The rest of our project uses `pass`, so this is also a minor readability improvement.

Reply:
Sure, no problem.